packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -308,6 +329,24 @@ export class WorkspaceCommandContribution implements CommandContribution {
                 isEnabled: () => this.workspaceService.isMultiRootWorkspaceEnabled,
                 isVisible: uris => this.areWorkspaceRoots(uris) && this.workspaceService.saved
             }));
+
+            registry.registerCommand(WorkspaceCommands.COPY_PATH, {
+                execute: (event?: Event) => {
+                    const fileUri: URI | undefined = UriSelection.getUri(this.selectionService.selection);
+                    if (fileUri) {
+                        this.handleCopy(fileUri.path.toString());
```

Comment:
It should be platform specific, not normalized, as @kittaakos mentioned on the issue.

Reply:
Correct, it does not work on Windows. These are copied to the clipboard instead of the platform-specific paths:
 - `/c:/Users/kittaakos/dev/theia/logo/theia-logo-no-text-black.svg`
 - `logo/theia-logo-no-text-black.svg`